### ISSUE TITLE
Consider breaking out the Enforcement Steps into a Separate Document

### ISSUE THREAD

**Issue by emmairwin** [ContributorCovenant/contributor_covenant#910]
Thanks for this great work!

I wonder if you would consider breaking out the Enforcement Guidelines into a separate document.  For a few reasons:

1) Being right in the document lends tone feels to be about consequence, vrs a tool for building inclusive communities.
2) The consequences feel inflexible, or not open variation because they are right in this as governance.  For example I have issued warnings, but those people still interacted with others involved ( for example volunteers working together in a support forum, would find it hard not to interact); in this case how would I explain that I did not follow the CoC rules?
3) Having a document that describes enforcement, can become a guide for other things (like how to take a report, and what to expect afterwards).  [Example](https://www.mozilla.org/en-US/about/governance/policies/participation/reporting/).

I can probably just do this on a community-basis, but wanted to suggest as an overall change as well, and would be happy to contribute to what that might look like.

Thanks!

**Comment by CoralineAda** [comment]
I like the idea a lot, but I'm not sure it could be effectively implemented on our side. What if we made this as a suggestion on the CC site itself under "Using Contributor Covenant"?

**Comment by emmairwin** [comment]
I think that as a suggestion is a good idea, that way its both permission to do that, and a nudge to think about what specifics of the enforcement may or may not make sense in context of their community (like not interacting with others, and warning example I gave).

Thanks for responding!

**Comment by CoralineAda** [comment]
@emmairwin Would you like to take a pass at updating the language on the home page?

**Comment by CoralineAda** [comment]
@emmairwin When you have a chance, please take a look at the home page. I rewrote it this weekend and I think it addresses some long-standing confusion around adoption and enforcement. If you have more suggestions, please feel free to open another issue. And thank you so much!

**Comment by emmairwin** [comment]
I like how you have organized that section now, and the prompt for people to understand the importance of that discussion about enforcement, and the tone they want to set.   Thank you!

The only other thing that may be good in future, it to describe how the expected behaviors make this document also a great tool for promoting healthy communities, leading by example etc.  If you agree of course.

**Comment by CoralineAda** [comment]
I really like your suggestion. Is this something you'd be interested in collaborating on?

**Comment by emmairwin** [comment]
for sure!  I'll propose something a bit later this week.

**Comment by AevaOnline** [comment]
Chiming in because this is an important point - including the Enforcement Steps as part of the main CC 2.0 text has made it difficult for me to support adoption in orgs that I work with, and we tend to stay at 1.4 because of this.

I've observed that less-experienced leaders will tend to fall back to the text of the document, and in some cases feel as though they must abide by the text as-given. Making it explicitly demonstrative, rather than proscriptive, would definitely help. Also, I really like some of the recent changes, such as encouraging inclusive behaviours and prioritizing the needs of any injured party through supporting documentation!

**Comment by agen007niaga** [comment]
Ejen007niaga

**Comment by janderssonse** [comment]
I agree with others - "Enforcement"-details is an implementation detail which should be left out of the general conduct text.

**Comment by Chealer** [comment]
I generally agree with this request. Enforcement is project-specific, and what the document currently contains is unclear/immature/poor. I for one contribute to tens (hundreds?) of projects, and familiarizing myself with a code of conduct for each would already be little realistic, in particular if its size is doubled by superfluous details.

The current state makes the whole document look very immature. Since version 2, the structure contains distinct _Enforcement Responsibilities_, _Enforcement_ and _Enforcement Guidelines_ sections. Most content about enforcement is not in _Enforcement_.

@emmairwin: Please retitle to describe the issue and clarify. It is unclear what you mean by "Enforcement Steps" and whether this is about enforcement in general or just part of it.

**Comment by CoralineAda** [comment]
@Chealer I see that you've been going through and commenting on lots of issues in the Contributor Covenant repo that have to do with the structure and content of the document. While your attention is appreciated, we're not making any changes to CC2 in light of the upcoming release of v3 of the code of conduct. We've conducted surveys and interviews to gather feedback that has been folded into our work. 

So thanks again, and keep your eye out for the new version.

**Comment by Chealer** [comment]
Hi @CoralineAda ,
This does not request changes to version 2 in particular. Your work is appreciated, but I'm afraid the issue(s) described here reflect [the document as it currently stands in the default branch](https://github.com/EthicalSource/contributor_covenant/blob/release/CODE_OF_CONDUCT.md). If there are tickets about problems which are already solved, thanks in advance for pointing these out.
